source/user-manual/api/examples.rst

```diff
@@ -127,25 +127,30 @@ Code:
     # Disable insecure https warnings (for self-signed SSL certificates)
     urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 
+
     # Functions
-    def get_response(url, headers, verify=False):
+    def get_response(request_type, url, headers, verify=False):
         """Get API result"""
-        request_result = requests.get(url, headers=headers, verify=verify)
+        if request_type == 'GET':
+            request_result = requests.get(url, headers=headers, verify=verify)
+        elif request_type == 'POST':
+            request_result = requests.post(url, headers=headers, verify=verify)
```

Comment:
Since you tried to improve this (good job), I suggest covering all type of methods. We could use an `enum` to wrap and verify all the different methods and then:

```python
request_result = getattr(requests, request_method)(url, headers=headers, verify=verify)
```

Also, `POST` and `PUT` requests could have JSON bodies. We should cover this as well.